src/main/java/uk/nhs/ctp/resourceProvider/EncounterProvider.java

```diff
@@ -87,6 +89,15 @@ public Bundle getEncounterReport(
     addList(bundle, caseId);
     addCompositions(bundle, caseId);
 
+    bundle.setTotal(1);
+    for (var entry : bundle.getEntry()) {
+      var searchMode = entry.getResource().getResourceType() == ResourceType.Encounter
```

Comment:
Could you also check for the exact match to the encounterParam, in case we have any previous encounters which end up referenced (e.g. handovers)?

Reply:
Yep, good point, I've updated the condition!